src/modules/claim/claim.types.ts

```diff
@@ -1,4 +1,10 @@
-export const NATS_EXCHANGE_TOPIC = 'claim.exchange';
+export const NATS_EXCHANGE_TOPIC = 'claim-exchange';
+
+export enum ClaimRequestType {
+  CLAIM_ISSUANCE = 'credential-issued',
+  CLAIM_REJECTION = 'credential-rejected',
+  CLAIM_ENROLMENT_REQUEST = 'request-credential',
+}
```

Comment:
I suggest we align the names of the event names from Aries protocol to remind us to stay in line with the specifications where we can 😄 
```suggestion
/**
 * Types inspired by Aries Issue-Credential protocol {@link https://github.com/hyperledger/aries-rfcs/blob/main/features/0453-issue-credential-v2/README.md}
 */
export enum ClaimRequestType {
  CLAIM_ISSUANCE = 'issue-credential',
  CLAIM_REJECTION = 'reject-credential',
  CLAIM_REQUEST = 'request-credential',
}
```